### PR TITLE
Making test only run mocha, not lint; adding npmrc

### PR DESCRIPTION
**Pull request by AlexBrickPorch**
Making the test script only run mocha.  Our PR test agent already runs lint explicitly.

### CODE REVIEW

**Comment by AlexBrickPorch** [comment]
I might have the wrong repo here.  Is this supposed to be public?

**Comment by pwmckenna** [comment]
yeah, that's intentional

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+unsafe-perm = true
+cache-min = 31536000
+repository = http://npm.mgmt.porch.com
```
Comment (pwmckenna) [inline .npmrc]: http://npmjs.com ?

Comment (AlexBrickPorch) [inline .npmrc]: Updated.

**Pull request merged**